src/main/java/microsoft/exchange/webservices/data/dns/DnsClient.java

```diff
@@ -43,6 +43,23 @@
 public class DnsClient {
 
   /**
+   * Set up the environment used to construct the DirContext.
+   *
+   * @param dnsServerAddress
+   * @return
+   */
+  static Hashtable<String, String> getEnv(String dnsServerAddress) {
```

Comment:
the method also should be reachable writing the tests in the correct test-package. In my opinion this one should be private/protected and maybe not static?